vg/draw/canvas.go

```diff
@@ -279,21 +317,65 @@ func (c *Canvas) Y(y float64) vg.Length {
 // Crop returns a new Canvas corresponding to the receiver
 // area with the given number of inches added to the minimum
 // and maximum x and y values of the Canvas's Rectangle.
-func (c Canvas) Crop(minx, miny, maxx, maxy vg.Length) Canvas {
+// Note that cropping the right and top sides of the canvas
+// requires specifying negative values of right and top.
+func (c Canvas) Crop(right, bottom, left, top vg.Length) Canvas {
 	minpt := Point{
-		X: c.Min.X + minx,
-		Y: c.Min.Y + miny,
+		X: c.Min.X + right,
+		Y: c.Min.Y + bottom,
 	}
 	maxpt := Point{
-		X: c.Max.X + maxx,
-		Y: c.Max.Y + maxy,
+		X: c.Max.X + left,
+		Y: c.Max.Y + top,
 	}
 	return Canvas{
-		Canvas:    vg.Canvas(c),
+		Canvas:    c,
 		Rectangle: Rectangle{Min: minpt, Max: maxpt},
 	}
 }
 
+// Tiles is a holder for subcanvases returned by the Tile function.
+type Tiles struct {
+	Values     []Canvas
+	Rows, Cols int
+}
+
+// At returns the canvas at the given row and column.
+func (t Tiles) At(row, col int) Canvas {
+	return t.Values[row*t.Cols+col]
+}
+
+// Tiles returns new Canvases corresponding to the reciever canvas tiled
+// into the given number of rows and columns (row major, starting from the
+// top left corner) where top, bottom, left, right, padx,
+// and pady are the padding above, below, to the left, to the right, between columns,
+// and between rows of the tiles, respectively.
+func (c Canvas) Tiles(rows, cols int, top, bottom, left, right, padx, pady vg.Length) *Tiles {
```

Comment:
There are a lot of parameters here. I'd like to avoid that. What if we restructured it as follows:
```
// Tiles specifies a tiling that can be applied to a Canvas.
// ... whatever else ...
type Tiles struct {
    // Cols and Raws specify the number of rows and columns of tiles.
    Cols, Rows int
    // PadTop, PadBottom, PadRight, and PadLeft specify the padding on the corresponding side of each tile.
    PadTop, PadBottom, PadRight, PadLeft vg.Length
    // PadX and PadY specify the padding between columns and rows of tiles respectively..
    PadX, PadY vg.Length
}

// At returns the subcanvas within c that corresponds to the tile at column x, row y.
type (ts Tiles) At(c Canvas, x, y int) Canvas { ... }
```
So, instead of:
```c.Tiles(5, 6, 0, 0, 0, 0, 1*vg.Millimeter, 1*vg.Milimeter)```
One would do:
```t := Tiles{ Rows: 5, Cols: 6, PadX: 1*vg.Millimeter, PadY: 1*vg.Millimeter }```
This is both more clear because the parameters are named, and it lets you take advantage of the zero-initialization.

Finally, it solves the issue of wanting both a type and a function named `Tiles`. Instead, you just have the type.